aes/host/main.c

```diff
@@ -37,6 +37,7 @@
 
 #define AES_TEST_BUFFER_SIZE	4096
 #define AES_TEST_KEY_SIZE	16
+#define AES_BLOCK_SIZE	16
```

Comment:
please add a indentation tabulation before `16`.

Reply:
Modified (and tested again)